source/appModules/explorer.py

```diff
@@ -178,6 +178,14 @@ def _get_value(self):
 			return value
 		return value.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
 
+class ReadOnlyEditBox(IAccessible):
+#Used for read-only edit boxes in a properties window.
+#These can contain dates that include unwanted left-to-right and right-to-left indicator characters.
+
+	def _get_windowText(self):
+		if  super(ReadOnlyEditBox, self).windowText is not None and controlTypes.STATE_READONLY in self.states:

```

Comment:
Same comment as with the replacements above: this code can be made a bit more readable.